migrations/versions/43e8c59337ae_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 43e8c59337ae
+Revises: 232b13d4e86a
+Create Date: 2019-06-18 10:13:37.254265
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '43e8c59337ae'
+down_revision = '232b13d4e86a'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('tickets', sa.Column('max_price', sa.Float(), server_default=0, nullable=True))
```

Comment:
This is nullable, so no default is recommended

Reply:
@iamareebjamal Done.